auth-web/src/util/config-helper.ts

```diff
@@ -235,7 +235,7 @@ export default class ConfigHelper {
   static paymentsAllowedPerAccountType () {
     return {
       [Account.BASIC]: [ PaymentTypes.CREDIT_CARD, PaymentTypes.ONLINE_BANKING ],
-      [Account.PREMIUM]: [ PaymentTypes.PAD, PaymentTypes.BCOL ],
+      [Account.PREMIUM]: [ PaymentTypes.PAD, PaymentTypes.BCOL, PaymentTypes.EFT ],
```

Comment:
this one need tweaks when api done "the flag the indicates the account is allowed to use EFT."

Reply:
There is already a call into payment_accounts somewhere.. , it should grab the enable_eft flag - put it in the store, if that's enabled it should provide this option to the client